index.html

```diff
@@ -276,8 +370,365 @@ <h4>RINGOVER</h4>
+          <p>LATEST</p>
+          <a href="#" class="comments-gear-a">CD Projekt Rekt: Cyberpunk 2077 tanks studio's </a>
+
+        </div>
+
+        <div class="gear-bottom">
+          <ul class="gear-list">
+            <li><a href="#" class="gear-bottom-a">Twitter shares fall 7% in pre-market trading after Trump's removal</a>
+            </li>
+            <li><a href="#" class="gear-bottom-a">Winklevoss comes out of meditation to witness Bitcoin's record
+                high</a></li>
+            <li><a href="#" class="gear-bottom-a">CD Projekt Rekt: Cyberpunk 2077 tanks studio's share price nearly
+                50%</a></li>
+            <li><a href="#" class="gear-bottom-a">CD Projekt stock down 20% since Cyberpunk reviews hit the internet</a>
+            </li>
+            <li><a href="#" class="gear-bottom-a">Airbnb to be bigger than eBay after its long-awaited IPO</a></li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="hard-fork">
+        <h2 class="title">Distract</h2>
+        <div class="gear-top1" id="top7">
+          <p>MOST POPULAR</p>
+          <a href="#" class="short">2021 will be the year open source projects overcome their diversity problems</a>
+          <p class="writer"><a href="#" class="writer">Ivan Mehta</a> 35 minutes gao</p>
+        </div>
+
+        <div class="gear-middle">
+          <p>LATEST</p>
+          <a href="#" class="comments-gear-a">Alibaba shares jump 8% as Jack Ma surfaces after 3 months</a>
+
+        </div>
+
+        <div class="gear-bottom">
+          <ul class="gear-list">
+            <li><a href="#" class="gear-bottom-a">Twitter shares fall 7% in pre-market trading after Trump's removal</a>
+            </li>
+            <li><a href="#" class="gear-bottom-a">Winklevoss comes out of meditation to witness Bitcoin's record
+                high</a></li>
+            <li><a href="#" class="gear-bottom-a">CD Projekt Rekt: Cyberpunk 2077 tanks studio's share price nearly
+                50%</a></li>
+            <li><a href="#" class="gear-bottom-a">CD Projekt stock down 20% since Cyberpunk reviews hit the internet</a>
+            </li>
+            <li><a href="#" class="gear-bottom-a">Airbnb to be bigger than eBay after its long-awaited IPO</a></li>
+          </ul>
+        </div>
+      </div>
     </section>
+
+    <section class="latest-container">
+      <div class="latest-cont-title">
+        <h2><a href="#" id="latest-deals-a">Latest deals ></a></h2>
+        <p><a href="#" id="deals-a">See All Deals</a></p>
+      </div>
+      <div class="latest">
+
+        <div class="latest1">
+          <a href="#"><img src="https://source.unsplash.com/random/250x185" alt="photo"></a>
+          <div class="content con">
+            <a href="#" class="img-text">
+              <p> The Ultimate 2021 White Hat Hacker Certification Bundle</p>
+            </a>
+            <p class="time"><span class="price">$13.45</span> $39.90</p>
+          </div>
+        </div>
```

Comment:
Each article in the `Latest Deals` section should have an image and a title (h1--h6) tag, both of them with a link (a tag). i.e.
```
<h4>
  <a href="#" class="img-text">
    <p> The Ultimate 2021 White Hat Hacker Certification Bundle</p>
  </a>
</h4>
```